benchmark_runner/common/oc/oc.py

```diff
@@ -48,6 +49,104 @@ def get_ocp_server_version(self):
         """
         return self.run(f"{self.__cli} get clusterversion version -o jsonpath='{{.status.desired.version}}'")
 
+    def upgrade_ocp(self, upgrade_ocp_version: str):
+        """
+        This method upgrades OCP version with conditional handling for specific versions.
+
+        @param upgrade_ocp_version: Version to upgrade to
+        @return:
+        """
+        ocp_channel = '.'.join(upgrade_ocp_version.split('.')[:2])
+        upgrade_command = f"{self.__cli} adm upgrade ; {self.__cli} adm upgrade channel stable-{ocp_channel} ; {self.__cli} adm upgrade --to={upgrade_ocp_version} --allow-explicit-upgrade;"
+
+        if ocp_channel == "4.16":
```

Comment:
Is this really a 4.16 flag day, or are you likely to have things like this happen with other releases?

Reply:
Yes, it special instruction for ocp 4.16.
add comment https://access.redhat.com/articles/7031404

Reply:
OK